src/tests/unit/tests/DetailsView/components/issues-details-pane.test.tsx

```diff
@@ -111,6 +112,9 @@ describe('IssuesDetailsPaneTest', () => {
         }
 
         const deps: IssuesDetailsPaneDeps = {
+            axeResultToIssueFilingDataConverter: {
```

Comment:
It would be better to mock this converter to return something meaningful. right now it renders undefined in the snapshot.